src/components/calcite-input/resources.ts

```diff
@@ -1,6 +1,8 @@
 export const CSS = {
   loader: "calcite-input__loader",
   clearButton: "calcite-input__clear-button",
+  editingEnabled: "calcite-input__editing-enabled",
```

Comment:
I think these are leftover from the work done to change inline-editable to not affect the input. Seems like these aren't being used though in this PR.